eZ/Bundle/EzPublishCoreBundle/ApiLoader/SearchEngineIndexerFactory.php

```diff
@@ -37,7 +37,7 @@ public function __construct(RepositoryConfigurationProvider $repositoryConfigura
     /**
      * Registers $searchEngineIndexer as a valid search engine indexer with identifier $searchEngineIdentifier.
      *
-     * @note It is strongly recommended to register indexer as a lazy service.
+     * note : It is strongly recommended to register indexer as a lazy service.
```

Comment:
Small nitpick: there shouldn't be a space before colon (or any punctuation for that matter - at least in English).